docker-compose.yaml

```diff
@@ -0,0 +1,52 @@
+
+version: '3'
+services:
+    sui-server:
+        image: sui-dev-image
+        init: true
+        ports:
+            - "4100:4100"
+            - "4200:4200"
+        volumes:
+            # base_app
+            - type: bind
+              source: "${PWD}/base_app"
+              target: /usr/src/app/base_app/
+            # sui-server
+            - type: bind
+              source: "${PWD}/simpleui-server"
+              target: /usr/src/app/simpleui-server/
+            # install-script.bash
+            - type: bind
+              source: "${PWD}/develop_docker/install-script.bash"
+              target: /usr/src/app/install-script.bash
+            # install-script-debug-simpleui-server.bash
+            - type: bind
+              source: "${PWD}/develop_docker/install-script-debug-simpleui-server.bash"
+              target: /usr/src/app/install-script-debug-simpleui-server.bash
```

Comment:
rather than mounting individual files you can just mount the whole directory, for example mounting `${PWD}/develop_docker/` to `/usr/src/develop_docker` or something

Reply:
changed docker-compose to mount the entire develop-docker folder instead of just those two install scripts